### PR TITLE
[FW-10929] feat(datepicker-improvement):  Added different month, year…

### PR DESCRIPTION
**Pull request by mikolajgrzyb**
… selection. Added today button.
Screen Shot 2021-07-26 at 4 56 01 PM

Please keep in mind this project has different standards.

### CODE REVIEW

File: dist/flatpickr.css [hunk 1]
```diff
@@ -758,6 +770,104 @@ span.flatpickr-weekday {
 .flatpickr-input[readonly] {
   cursor: pointer;
 }
+.flatpickr-today-button {
```
Comment (mikolajgrzyb) [inline dist/flatpickr.css:773]: Look in this file.

**Review by MagdaMalinowska — APPROVED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -928,49 +929,53 @@ function FlatpickrInstance(
...
+      self.currentYear === self.config.maxDate.getFullYear() &&
+      month > self.config.maxDate.getMonth()
+    );
+  }
 
+  function buildMonthOptions(): { value: string; label: string }[] {
+    const options = [];
     for (let i = 0; i < 12; i++) {
       if (!shouldBuildMonth(i)) continue;
+      const value = new Date(self.currentYear, i).getMonth().toString();
```
Comment (pawelofieldly) [inline src/index.ts]: local vars type def missing

File: src/index.ts [hunk 3]
```diff
@@ -928,49 +929,53 @@ function FlatpickrInstance(
...
+      const label = monthToStr(i, self.config.shorthandCurrentMonth, self.l10n);
```
Comment (pawelofieldly) [inline src/index.ts]: too

**Review by pawelofieldly — APPROVED** [review]
Approved, one thing I noticed that you doesn't type local variables.

**Pull request merged**